src/generators/static/tsConfigFileGenerator.ts

```diff
@@ -34,7 +34,7 @@ const restLevelTsConfigInAzureSdkForJs = {
   "include": ["src/**/*.ts"]
 };
 
-const restLevelTsConfigNotInAzureSdkForJs ={
+const restLevelTsConfigNotInAzureSdkForJs: Record<string, any> = {
```

Comment:
I wonder if there is a type for this that we can import from the typescript package

Reply:
I didn't find this `type` after searching it. Let me know if you know similar types.